subsys/bindesc/Kconfig

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2023 Yonatan Schachter
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig BINDESC
+	bool "Binary Descriptors"
+	depends on ARCH_SUPPORTS_ROM_START || BOARD_NATIVE_POSIX
+	help
+	  Binary Descriptors - constant data accessible outside of the executable image
+
+if BINDESC
+
+config BINDESC_DEFINE
+	bool "Binary Descriptors Define"
+	help
+	  Enable the app to define its own binary descriptors
+
+if BINDESC_DEFINE
```

Comment:
Shouldn't this not be here, as version, build time, etc. would not be defining own descriptors, they are in-built ones?

Reply:
This symbol is to differentiate between defining descriptors and reading them (in the future). For example, the bootloader would need to read the descriptors of the app without defining its own, and the app would define descriptors without needing to read any.